node_modules/.bin/acorn

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Please, add everything from node_modules to .gitignore. Also remove this files from repository